kolibri/plugins/learn/assets/src/views/SearchFiltersPanel/index.vue

```diff
@@ -365,14 +365,15 @@
   }
 
   .side-panel {
+    @extend %dropshadow-2dp;
```

Comment:
There isn't very clear guidance for this on the design system documentation page, but I think we should update to `4dp` for consistency with the other navigation bar items